File: 1431.py
class Solution:
    def kidsWithCandies(self, candies: list[int], extraCandies: int) -> list[bool]:
        maxcandy = max(candies)
        maxes: list[bool] = []
        for i in range(len(candies)):
            if (candies[i] + extraCandies) >= maxcandy:
                maxes.append(True)
            else:
                maxes.append(False)
        return maxes

File: 1431.ts
function kidsWithCandies(candies: number[], extraCandies: number): boolean[] {
  let max: number = 0;
  let maxes: boolean[] = [];
  candies.forEach((e) => {
    if (e > max) {
      max = e;
    }
  });
  for (let i = 0; i < candies.length; i++) {
    if (candies[i] + extraCandies >= max) {
      maxes.push(true);
    } else {
      maxes.push(false);
    }
  }
  return maxes;
}
